src/app/modules/courses/components/course-card/course-card.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="content">
+  <div class="time">
+    <div>{{course.duration}}</div>
+    <div>{{course.creationDate | date}}</div>
+  </div>
+
+  <h2>{{course.title}}</h2>
+  <div class="description-wrapper">
+    <div class="description">
+      {{course.description}}
+    </div>
+    <div class="action">
+      <div><button (click)="edit.emit()">Edit</button></div>
```

Comment:
no need to wrap button in div, if you need each button on new line, just add proper styles

Reply:
fixed